modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_entries_admin/account_role/details.jsp

```diff
@@ -34,6 +34,7 @@ if (accountRole != null) {
 
 <liferay-frontend:edit-form
 	action="<%= editAccountRoleURL %>"
+	cssClass="container-form-lg"
```

Comment:
There are like 100 usages of `liferay-frontend:edit-form`. Why do only these need the additional class?

Reply:
I wasn't able to make a blanket change by outputting `container-form-lg` by default since some `liferay-frontend:edit-form` tags are inside a modal.

We also have a lot of places where we still have old markup like `card-horizontal main-content-card` which adds vertical space. I think we should take it on a case by case basis as we replace the old components.

We could have it output `container-form-lg` by default and deal with the consequences later.